### ISSUE TITLE
Dialog docs do not mention required nuget package

### ISSUE THREAD

**Issue by navzam** [MicrosoftDocs/bot-docs#995]
The [first doc on using dialogs](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-dialog-manage-conversation-flow?view=azure-bot-service-4.0&tabs=csharp) does not mention that you need to install the `Microsoft.Bot.Builder.Dialogs` package to use dialogs. As a result, if you try to use the docs to implement dialogs in your own bot, then you end up with "red squiggles" in Visual Studio. The only way you would know to install the nuget package is by digging into the `.csproj` of the referenced sample.

This is essentially a duplicate of #633, which was closed, but this is still an issue.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 04c5adf9-272a-e0f1-10f1-c8365e76035b
* Version Independent ID: 1982af41-5a06-4398-0e95-abcc50864e0d
* Content: [Implement sequential conversation flow - Bot Service](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-dialog-manage-conversation-flow?view=azure-bot-service-4.0&tabs=csharp)
* Content Source: [articles/v4sdk/bot-builder-dialog-manage-conversation-flow.md](https://github.com/MicrosoftDocs/bot-docs/blob/live/articles/v4sdk/bot-builder-dialog-manage-conversation-flow.md)
* Service: **bot-service**
* Sub-service: **sdk**
* GitHub Login: @JonathanFingold
* Microsoft Alias: **v-jofing**

**Comment by JonathanFingold** [comment]
We have an update for this topic, but it's waiting on the merging of a branch in the botbuilder-samples repo. The merge looks like it's still over a week out, but for that doc push, every develop topic should explicitly call out the NuGet or npm packages used in the sample that are not included by default in the VSIX or Yeoman template.

**Comment by JonathanFingold** [comment]
The samples and how-to articles have been updated. These nmp/NuGet package references should all be good now.
